### ISSUE TITLE
Some updates and errors

### ISSUE THREAD

**Issue by paulocoutinhox** [TheOctan/Shootetar#1]
Hi,

Can you update to latest version?

**Some updates in manifest.json:**

From: `"com.unity.ai.navigation.components": "https://github.com/Unity-Technologies/NavMeshComponents.git#package",`
To: `"com.unity.ai.navigation": "2.0.0",`

Also, add: `"com.unity.nuget.newtonsoft-json": "3.2.1",`

**Some erros:**

```
Could not create asset from Assets/Gizmos/RainbowFolders/64x64/ControlsFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/GizmosFolder Color Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/VersionControlIgnoreFolder Color Icon.png: File could not be read

Could not create asset from Assets/Gizmos/OctanGames/Managers/MusicManager Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/OctanGamesFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/SpritesFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/OctanGames/Receivers/ScoreReceiver Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/PresetFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/GizmosFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/ArtFolder Color Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/SandBoxFolder Color Icon.png: File could not be read

Could not create asset from Assets/Project/Sprites/Background.jpg: File could not be read

Could not create asset from Assets/Project/Sprites/Muzzleflash.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/ArtFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/OctanGames/Props/Projectile Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/SandBoxFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/OctanGames/Props/Crosshairs Icon.png: File could not be read

Could not create asset from Assets/Project/Sprites/Icon.psd: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/ControlsFolder Color Icon.png: File could not be read

Could not create asset from Assets/Gizmos/OctanGames/Controllers/MovementController Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/SpritesFolder Color Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/SpritesFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/OctanGames/LockRotation Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/ThirdPartyAssetsFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/OctanGames/Managers/AudioManager Icon.png: File could not be read

Could not create asset from Assets/Project/Sprites/Shotgun.png: File could not be read

Could not create asset from Assets/Gizmos/OctanGames/Props/Gun Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/StreamingAssetsFolder Color Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/ThirdPartyAssetsFolder Color Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/DocumentationFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/VersionControlIgnoreFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/OctanGames/Props/Shell Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/ArtFolder Color Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/PresetFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/ControlsFolder Color Icon.png: File could not be read

Could not create asset from Assets/Project/Sprites/Shotgun_Gradation.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/OctanGamesFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/ThirdPartyAssetsFolder Color Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/DocumentationFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/SandBoxFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/SpritesFolder Color Icon.png: File could not be read

Could not create asset from Assets/Project/Sprites/Crosshairs.png: File could not be read

Could not create asset from Assets/Project/Sprites/Game Logo.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/ControlsFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/StreamingAssetsFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/GizmosFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/OctanGames/UI/UIBillboard Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/VersionControlIgnoreFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/OctanGames/Entities/Player/PlayerController Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/ThirdPartyAssetsFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/StreamingAssetsFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/SandBoxFolder Color Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/16x16/ArtFolder Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/StreamingAssetsFolder Color Icon.png: File could not be read

Could not create asset from Assets/Gizmos/OctanGames/Controllers/GunController Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/PresetFolder Color Icon.png: File could not be read

Could not create asset from Assets/Gizmos/RainbowFolders/64x64/GizmosFolder Color Icon.png: File could not be read

The Progressive CPU lightmapper is not supported on Apple silicon, switching to the Progressive GPU lightmapper.

FreeType error: The font Assets/Project/Fonts/OpenSans-Regular.ttf could not be imported because the  format is unsupported

Blender could not be found.
Make sure that Blender is installed and the .blend file has Blender as its 'Open with' application!

Errors during import of AudioClip Assets/Project/Audio/Music/Menu theme - Thiago Adamo.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Guns/Gun 03_shoot.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Impacts/Impact_01.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Guns/Gun 04_shoot.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Guns/Gun 02_reload.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Enemy/Enemy death_02.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Player/Level completed.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Impacts/Impact_03.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Guns/Gun 02_shoot.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Guns/Gun 03_reload.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Guns/Gun 04_reload.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Guns/Gun 01_reload.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Enemy/Enemy death_01.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Music/Main theme - Thiago Adamo.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Guns/Gun 05_shoot.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Guns/Gun 01_shoot.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Enemy/Enemy attack.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Impacts/Impact_02.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Player/Player death.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.


Errors during import of AudioClip Assets/Project/Audio/Guns/Gun 05_reload.wav:
FSBTool ERROR: The format of the source file is invalid, see output for details.
FSBTool ERROR: Internal error from FMOD sub-system.
```

**Comment by TheOctan** [comment]
Hello, I fix it and update project to Unity 2022.3.14

**Comment by TheOctan** [comment]
Hello, I fix it and update project to Unity 2022.3.14

**Comment by paulocoutinhox** [comment]
Hi,

It still with problems.

Problem 1 - Deprecated plugin

<img width="1162" alt="Screenshot 2023-12-27 at 14 29 02" src="https://github.com/TheOctan/ShotGun/assets/395096/3b98e2b1-e5ed-4213-a699-27f07bbeacce">

Problem 2 - Assets not found and code errors

```
Could not create asset from Assets/Project/Sprites/Muzzleflash.png: File could not be read

Could not create asset from Assets/TextMesh Pro/Sprites/EmojiOne.png: File could not be read

FreeType error: The font Assets/TextMesh Pro/Fonts/LiberationSans.ttf could not be imported because the  format is unsupported

The Progressive CPU lightmapper is not supported on Apple silicon, switching to the Progressive GPU lightmapper.
UnityEditor.EditorApplication:Internal_RestoreLastOpenedScenes ()

Assets/Standard Assets/Effects/ImageEffects/Scripts/ImageEffectBase.cs(20,18): warning CS0618: 'SystemInfo.supportsImageEffects' is obsolete: 'supportsImageEffects always returns true, no need to call it'

Assets/Standard Assets/Effects/ImageEffects/Scripts/PostEffectsBase.cs(97,18): warning CS0618: 'SystemInfo.supportsImageEffects' is obsolete: 'supportsImageEffects always returns true, no need to call it'

Assets/Standard Assets/Effects/ImageEffects/Scripts/PostEffectsBase.cs(97,54): warning CS0618: 'SystemInfo.supportsRenderTextures' is obsolete: 'supportsRenderTextures always returns true, no need to call it'

Assets/Standard Assets/Effects/ImageEffects/Scripts/ScreenSpaceAmbientOcclusion.cs(64,18): warning CS0618: 'SystemInfo.supportsImageEffects' is obsolete: 'supportsImageEffects always returns true, no need to call it'

Assets/Standard Assets/Modern UI Pack/Scripts/Window/WindowDragger.cs(24,42): warning CS0618: 'Object.FindObjectsOfType(Type)' is obsolete: 'Object.FindObjectsOfType has been deprecated. Use Object.FindObjectsByType instead which lets you decide whether you need the results sorted or not.  FindObjectsOfType sorts the results by InstanceID, but if you do not need this using FindObjectSortMode.None is considerably faster.'

Assets/Standard Assets/Modern UI Pack/Scripts/Notification/NotificationManager.cs(60,58): warning CS0618: 'Object.FindObjectsOfType(Type)' is obsolete: 'Object.FindObjectsOfType has been deprecated. Use Object.FindObjectsByType instead which lets you decide whether you need the results sorted or not.  FindObjectsOfType sorts the results by InstanceID, but if you do not need this using FindObjectSortMode.None is considerably faster.'

Assets/Project/Scripts/Map/Spawner.cs(43,18): warning CS0618: 'Object.FindObjectOfType<T>()' is obsolete: 'Object.FindObjectOfType has been deprecated. Use Object.FindFirstObjectByType instead or if finding any instance is acceptable the faster Object.FindAnyObjectByType'

Assets/Project/Scripts/Map/Spawner.cs(50,9): warning CS0618: 'Object.FindObjectOfType<T>()' is obsolete: 'Object.FindObjectOfType has been deprecated. Use Object.FindFirstObjectByType instead or if finding any instance is acceptable the faster Object.FindAnyObjectByType'

Assets/Project/Scripts/Map/Spawner.cs(80,27): warning CS0618: 'Object.FindObjectsOfType<T>()' is obsolete: 'Object.FindObjectsOfType has been deprecated. Use Object.FindObjectsByType instead which lets you decide whether you need the results sorted or not.  FindObjectsOfType sorts the results by InstanceID but if you do not need this using FindObjectSortMode.None is considerably faster.'

Assets/Project/Scripts/UI/GameUI.cs(106,14): warning CS0618: 'Object.FindObjectOfType<T>()' is obsolete: 'Object.FindObjectOfType has been deprecated. Use Object.FindFirstObjectByType instead or if finding any instance is acceptable the faster Object.FindAnyObjectByType'

Assets/Project/Scripts/UI/GameUI.cs(111,13): warning CS0618: 'Object.FindObjectOfType<T>()' is obsolete: 'Object.FindObjectOfType has been deprecated. Use Object.FindFirstObjectByType instead or if finding any instance is acceptable the faster Object.FindAnyObjectByType'

Assets/Project/Scripts/Experimental/Managers/SessionManager.cs(22,14): warning CS0618: 'Object.FindObjectOfType<T>()' is obsolete: 'Object.FindObjectOfType has been deprecated. Use Object.FindFirstObjectByType instead or if finding any instance is acceptable the faster Object.FindAnyObjectByType'

Assets/Project/Scripts/Experimental/Managers/SessionManager.cs(24,10): warning CS0618: 'Object.FindObjectOfType<T>()' is obsolete: 'Object.FindObjectOfType has been deprecated. Use Object.FindFirstObjectByType instead or if finding any instance is acceptable the faster Object.FindAnyObjectByType'

Assets/Project/Scripts/Experimental/Managers/SessionManager.cs(66,10): warning CS0618: 'Object.FindObjectOfType<T>()' is obsolete: 'Object.FindObjectOfType has been deprecated. Use Object.FindFirstObjectByType instead or if finding any instance is acceptable the faster Object.FindAnyObjectByType'

Assets/Project/Scripts/Managers/AudioManager.cs(103,21): warning CS0618: 'Object.FindObjectOfType<T>()' is obsolete: 'Object.FindObjectOfType has been deprecated. Use Object.FindFirstObjectByType instead or if finding any instance is acceptable the faster Object.FindAnyObjectByType'

Assets/Project/Scripts/Managers/AudioManager.cs(104,18): warning CS0618: 'Object.FindObjectOfType<T>()' is obsolete: 'Object.FindObjectOfType has been deprecated. Use Object.FindFirstObjectByType instead or if finding any instance is acceptable the faster Object.FindAnyObjectByType'

Assets/Project/Scripts/Managers/AudioManager.cs(135,9): warning CS0618: 'Object.FindObjectOfType<T>()' is obsolete: 'Object.FindObjectOfType has been deprecated. Use Object.FindFirstObjectByType instead or if finding any instance is acceptable the faster Object.FindAnyObjectByType'

Assets/Project/Scripts/Managers/AudioManager.cs(137,16): warning CS0618: 'Object.FindObjectOfType<T>()' is obsolete: 'Object.FindObjectOfType has been deprecated. Use Object.FindFirstObjectByType instead or if finding any instance is acceptable the faster Object.FindAnyObjectByType'

Assets/Standard Assets/Modern UI Pack/Editor/ContextMenu.cs(18,42): warning CS0618: 'Object.FindObjectsOfType(Type)' is obsolete: 'Object.FindObjectsOfType has been deprecated. Use Object.FindObjectsByType instead which lets you decide whether you need the results sorted or not.  FindObjectsOfType sorts the results by InstanceID, but if you do not need this using FindObjectSortMode.None is considerably faster.'

Assets/Standard Assets/Modern UI Pack/Editor/ContextMenu.cs(36,38): warning CS0618: 'Object.FindObjectsOfType(Type)' is obsolete: 'Object.FindObjectsOfType has been deprecated. Use Object.FindObjectsByType instead which lets you decide whether you need the results sorted or not.  FindObjectsOfType sorts the results by InstanceID, but if you do not need this using FindObjectSortMode.None is considerably faster.'

Assets/Standard Assets/Modern UI Pack/Editor/ContextMenu.cs(53,42): warning CS0618: 'Object.FindObjectsOfType(Type)' is obsolete: 'Object.FindObjectsOfType has been deprecated. Use Object.FindObjectsByType instead which lets you decide whether you need the results sorted or not.  FindObjectsOfType sorts the results by InstanceID, but if you do not need this using FindObjectSortMode.None is considerably faster.'

Assets/Standard Assets/Modern UI Pack/Editor/ContextMenu.cs(71,38): warning CS0618: 'Object.FindObjectsOfType(Type)' is obsolete: 'Object.FindObjectsOfType has been deprecated. Use Object.FindObjectsByType instead which lets you decide whether you need the results sorted or not.  FindObjectsOfType sorts the results by InstanceID, but if you do not need this using FindObjectSortMode.None is considerably faster.'

Assets/Standard Assets/Effects/ImageEffects/Editor/ColorCorrectionLookupEditor.cs(54,25): warning CS0618: 'TextureImporter.textureFormat' is obsolete: 'textureFormat is no longer accessible at the TextureImporter level. For old 'simple' formats use the textureCompression property for the equivalent automatic choice (Uncompressed for TrueColor, Compressed and HQCommpressed for 16 bits). For platform specific formats use the [[PlatformTextureSettings]] API. Using this setter will setup various parameters to match the new automatic system as well as possible. Getter will return the last value set.'

Assets/Standard Assets/Effects/ImageEffects/Editor/ColorCorrectionLookupEditor.cs(54,58): error CS0619: 'TextureImporterFormat.AutomaticTruecolor' is obsolete: 'Enum member TextureImporterFormat.AutomaticTruecolor is obsolete. Use the TextureImporter.textureCompression property instead.'

Assets/Standard Assets/Effects/ImageEffects/Editor/ColorCorrectionLookupEditor.cs(62,25): warning CS0618: 'TextureImporter.textureFormat' is obsolete: 'textureFormat is no longer accessible at the TextureImporter level. For old 'simple' formats use the textureCompression property for the equivalent automatic choice (Uncompressed for TrueColor, Compressed and HQCommpressed for 16 bits). For platform specific formats use the [[PlatformTextureSettings]] API. Using this setter will setup various parameters to match the new automatic system as well as possible. Getter will return the last value set.'

Assets/Standard Assets/Effects/ImageEffects/Editor/ColorCorrectionLookupEditor.cs(62,57): error CS0619: 'TextureImporterFormat.AutomaticTruecolor' is obsolete: 'Enum member TextureImporterFormat.AutomaticTruecolor is obsolete. Use the TextureImporter.textureCompression property instead.'
```

Thanks.
